lib/engine/g_1856/corporation.rb

```diff
@@ -56,6 +55,28 @@ def release_escrow!
         @capitalization = :incremental
       end
 
+      # Issue more shares
+      def issue_shares!
```

Comment:
should be in the game class and not in corporation

Reply:
This is now in `g_1856.rb` as `national_issue_shares!`